services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/service/VpcService.java

```diff
@@ -2,12 +2,15 @@
 
 import com.futurewei.alcor.vpcmanager.exception.SubnetsNotEmptyException;
 import com.futurewei.alcor.web.entity.route.RouteWebJson;
+import com.futurewei.alcor.web.entity.route.RouterWebJson;
 import com.futurewei.alcor.web.entity.vpc.VpcEntity;
 
 public interface VpcService {
 
     public RouteWebJson getRoute (String vpcId, VpcEntity vpcState);
 
+    public RouterWebJson getRoute_test (String routerId, String routeTableId, VpcEntity vpcState);
```

Comment:
what is getRoute_test for? Something like "_test" is usually not recommended in the interface.

Reply:
Actually this is for the test of new version of route manager, I will remove it